internal/graph/resourcesubjectsmap.go

```diff
@@ -104,6 +93,25 @@ func (rsm dispatchableResourcesSubjectMap) resourceIDs() []string {
 	return rsm.resourcesAndSubjects.Keys()
 }
 
+// filterSubjectIDsToDispatch returns the set of subject IDs that have not yet been
+// dispatched, by adding them to the dispatched set.
+func (rsm dispatchableResourcesSubjectMap) filterSubjectIDsToDispatch(dispatched *syncONRSet, dispatchSubjectType *core.RelationReference) []string {
+	resourceIDs := rsm.resourceIDs()
+	filtered := make([]string, 0, len(resourceIDs))
+	for _, resourceID := range resourceIDs {
+		if dispatched.Add(&core.ObjectAndRelation{
+			Namespace: dispatchSubjectType.Namespace,
+			ObjectId:  resourceID,
+			Relation:  dispatchSubjectType.Relation,
+		}) {
+			filtered = append(filtered, resourceID)
+			continue
```

Comment:
maybe I'm missing something but is `continue` necessary here?

Reply:
Removed. This was from a previous version